lke_node_pools.go

```diff
@@ -47,7 +47,8 @@ type LKENodePool struct {
 	Linodes []LKENodePoolLinode `json:"nodes"`
 	Tags    []string            `json:"tags"`
 
-	Autoscaler LKENodePoolAutoscaler `json:"autoscaler"`
+	Autoscaler     LKENodePoolAutoscaler  `json:"autoscaler"`
+	DiskEncryption InstanceDiskEncryption `json:"disk_encryption,omitempty"`
```

Comment:
Based on Epic non-functional requirements it should not be possible to manage DiskEncryption by user. So we should not expose this on LKENodePool.